services/QuillLMS/client/app/bundles/Staff/components/activityForm/topicColumn.tsx

```diff
@@ -65,12 +55,20 @@ const TopicColumn = ({ createNewTopic, levelNumber, getFilteredOptionsForLevel,
     <div className="topic-column">
       <label>Topic Level {levelNumber}</label>
       {selectedOptionElement}
-      <ReactTable
-        columns={columns(selectedOption)}
-        data={options}
-        defaultFilterMethod={(filter, row) => row._original.name ? row._original.name.toLowerCase().includes(filter.value.toLowerCase()) : ''}
-        filterable
-      />
+      <div>
+        {
+          options.map((option, i) => {
+            let clickEventHelper = () => {
+              handleClickSelectTopic(option.id)
+            };
+            return (
+              <button className="record-cell" key={i} onClick={clickEventHelper} onKeyDown={clickEventHelper} type="button">
```

Comment:
You shouldn't need an `onKeyDown` for a `button` element (it will automatically be handled by the `onClick`, but that isn't true for interactive elements).